syne_tune/optimizer/schedulers/searchers/gp_fifo_searcher.py

```diff
@@ -107,7 +108,8 @@ def _create_internal(
             num_initial_random_choices: int = DEFAULT_NUM_INITIAL_RANDOM_EVALUATIONS,
             initial_scoring: Optional[str] = None,
             cost_attr: Optional[str] = None,
-            resource_attr: Optional[str] = None):
+            resource_attr: Optional[str] = None,
+            filter_observed_data: Optional[ConfigurationFilter] = None):
```

Comment:
Can this one be added to the docstring? (I could not find anywhere)

Reply:
I added something to transfer_learning_task_attr docstring. The thing is this filter_observed_data is not something you set directly.